### DISCUSSION TITLE
Cannot build: Qt5LinguistTools is missing

### DISCUSSION THREAD

**Discussion by PrSunflower** [KDE/ghostwriter#849] [Q&A]
Hello fellow community :slightly_smiling_face: 

I am building Ghostwriter from source but when running **cmake** it runs into an error:

```
user@PC:~/Builds/ghostwriter/build$ cmake ..
-- The CXX compiler identification is GNU 12.2.0
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Installing in the same prefix as Qt, adopting their path scheme.
-- Setting build type to 'Debug' as none was specified.
-- Looking for __GLIBC__
-- Looking for __GLIBC__ - found
-- Performing Test _OFFT_IS_64BIT
-- Performing Test _OFFT_IS_64BIT - Success
-- Performing Test HAVE_DATE_TIME
-- Performing Test HAVE_DATE_TIME - Success
CMake Error at /usr/share/ECM/modules/ECMPoQmTools.cmake:149 (find_package):
  Could not find a package configuration file provided by "Qt5LinguistTools"
  with any of the following names:

    Qt5LinguistToolsConfig.cmake
    qt5linguisttools-config.cmake

  Add the installation prefix of "Qt5LinguistTools" to CMAKE_PREFIX_PATH or
  set "Qt5LinguistTools_DIR" to a directory containing one of the above
  files.  If "Qt5LinguistTools" provides a separate development package or
  SDK, be sure it has been installed.
Call Stack (most recent call first):
  /usr/share/ECM/modules/ECMPoQmTools.cmake:242 (ecm_process_po_files_as_qm)
  CMakeLists.txt:46 (ecm_install_po_files_as_qm)


-- Configuring incomplete, errors occurred!
See also "/home/user/Builds/ghostwriter/build/CMakeFiles/CMakeOutput.log".
```

- I have followed the steps to "have the necessary packages installed for Qt 5 and KDE Frameworks"

- I am running **Debian Testing**, currently on 12 'bookworm'

<br>

I would be grateful if someone could assist me :slightly_smiling_face:

**Comment by bobhairgrove** [comment]
Looks like you also need to install 'qt5tools-dev' (I just had the same problem today on Ubuntu 22.04):

https://stackoverflow.com/questions/51698075/cmake-cannot-find-qt5linguisttools-in-docker-ubuntu-18-04

**Reply by PrSunflower** [reply]
Hi, apologies for the delay :slightly_smiling_face: 

Thank you for your help! wereturtle is also helping me here: https://bugs.kde.org/show_bug.cgi?id=468255

She has suggested the same but I am having another error now. I'll be troubleshooting on KDE Bugs and I will post here when we find a solution :slightly_smiling_face: 

Thanks, see you :wave:
